integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDataFrameWriter.scala

```diff
@@ -34,12 +34,37 @@ class CarbonDataFrameWriter(sqlContext: SQLContext, val dataFrame: DataFrame) {
   def saveAsCarbonFile(parameters: Map[String, String] = Map()): Unit = {
     // create a new table using dataframe's schema and write its content into the table
     sqlContext.sparkSession.sql(makeCreateTableString(dataFrame.schema,
-    new CarbonOption(parameters)))
+      new CarbonOption(parameters)))
     writeToCarbonFile(parameters)
   }
 
-  def appendToCarbonFile(parameters: Map[String, String] = Map()): Unit = {
-    writeToCarbonFile(parameters)
+  private def makeCreateTableString(schema: StructType, options: CarbonOption): String = {
```

Comment:
It seems this function is moved from line 144 to here? It is better not to move it so that it is easier to review